Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:latest
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 3003
+
+CMD ["npm","run","server"]
```

Comment:
I think we will need to update this to npm start

Reply:
yes